governance.md

```diff
@@ -139,13 +139,14 @@ In particular, the Council may:
     project.
 -   Make decisions about strategic collaborations with other organizations or
     individuals.
--   Make decisions about specific technical issues, features, bugs and pull
```

Comment:
What is the rationale for this deletion?  Is it that the item should apply to anyone with commit rights?  Or is the point to avoid diluting the importance of "consensus"?

Reply:
My interpretation is that such decisions come about from the consensus of people with commit rights, not the steering council.

Reply:
Yes.  Steering council should be about which way to go and the management around getting there but should stay out of day-to-day.